From 73660f4b548e50fdb6f411b4f4f0646693deae7e Mon Sep 17 00:00:00 2001
From: Chin-Ting Kuo <chin-ting_kuo@aspeedtech.com>
Date: Tue, 31 Mar 2026 13:57:48 +0800
Subject: [PATCH] dts: arm: ast2700: Fix bug for SPI2 Quad IO settings

"SPI2QUAD" function or group must be used for enabling SPI2 Quad IO.

Signed-off-by: Chin-Ting Kuo <chin-ting_kuo@aspeedtech.com>
Change-Id: I844dd19f4a398c2095084d4be90f68923981b59b
(cherry picked from commit 800c55b0ca079abd6d9fd7eb873a520602493b72)

Signed-off-by: santhoshs <santhoshs@ami.com>
Signed-off-by: Manoj Ashok <amanoj@ami.com>
---
 arch/arm/dts/ast2700.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/dts/ast2700.dtsi b/arch/arm/dts/ast2700.dtsi
index 517d028525..8664729437 100644
--- a/arch/arm/dts/ast2700.dtsi
+++ b/arch/arm/dts/ast2700.dtsi
@@ -1381,8 +1381,8 @@
 	};
 
 	pinctrl_spi2_quad_default: spi2_quad_default {
-		function = "SPI1QUAD";
-		groups = "SPI1QUAD";
+		function = "SPI2QUAD";
+		groups = "SPI2QUAD";
 	};
 
 	pinctrl_sdio_default: sdio_default {
-- 
2.34.1

